yellowbrick/text/dispersion.py

```diff
@@ -79,13 +103,14 @@ def _compute_dispersion(self, text):
                 # In the case that word is not in target words, any empty list is
                 # returned and no data will be yielded
                 self.offset += 1
-                for y in (self.target_words_ == word).nonzero()[0]:
-                    yield (self.offset, y)
+                for y_coord in (self.target_words_ == word).nonzero()[0]:
+                    y_coord = int(y_coord)
+                    yield (self.offset, y_coord, target)
             if self.annotate_docs:
                 self.boundaries_.append(self.offset)
         self.boundaries_ = np.array(self.boundaries_, dtype=int)
 
-    def fit(self, text):
+    def fit(self, text, y=None, **kwargs):
```

Comment:
Would you mind if we changed `text` to `X` to make it a bit more similar to how input features are referenced in scikit-learn and in our other text visualizers (e.g. [FreqDist](http://www.scikit-yb.org/en/develop/_modules/yellowbrick/text/freqdist.html#FrequencyVisualizer.fit) and [TSNE](http://www.scikit-yb.org/en/develop/_modules/yellowbrick/text/tsne.html#TSNEVisualizer.fit)?